### PR TITLE
Allowed paths implementation

### PR DESCRIPTION
**Pull request by sapajune**

### CODE REVIEW

File: cfgfile/schema.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type SidecarConfig struct {
 	UpdateInterval  int      `config:"update_interval"`
 	SendStatus      bool     `config:"send_status"`
 	ListLogFiles    []string `config:"list_log_files"`
+	AllowedPaths		[]string `config:"allowed_paths"`
```
Comment (kompa3) [inline cfgfile/schema.go]: Please add allowed_paths to Configuration section of README.md

Comment (sapajune) [reply cfgfile/schema.go]: Already have it but not committed here.

**Comment by sapajune** [comment]
I guess code could have been more understandable if it was not necessary to work with loosely typed filebeat configuration.

File: cfgfile/schema.go (hunk 1 above)
Comment (sapajune) [reply cfgfile/schema.go]: Added description.

Comment (kompa3) [reply cfgfile/schema.go]: The documentation does not explain that the setting is regexp.

**Pull request merged**